orchestra/interface_api/project_management/views.py

```diff
@@ -131,3 +133,17 @@ def end_project_api(request):
         end_project(project_id)
     except Project.DoesNotExist:
         raise BadRequest('Project not found for the given id.')
+
+
+@project_management_api_view
+def set_project_status_api(request):
```

Comment:
if this superscedes end_project_api, then clean that up everywhere and ensure any tests related to end_project_api end up having their logic tested in tests for set_project_status_api